src/emulator/shaders/6959385632856496271.glsl

```diff
@@ -7,8 +7,16 @@ attribute vec2 aTexcoord;
 
 varying vec2 vTexcoord;
 
+mat4 transpose(mat4 m) {
+  return mat4(m[0][0], m[1][0], m[2][0], m[3][0],
+              m[0][1], m[1][1], m[2][1], m[3][1],
+              m[0][2], m[1][2], m[2][2], m[3][2],
+              m[0][3], m[1][3], m[2][3], m[3][3]);
+}
+
 void main()
 {
-    gl_Position = vec4(aPosition, 1) * wvp;
+    gl_Position = vec4(aPosition, 1) * transpose(wvp);
```

Comment:
What I would try here instead is reordering the multiplication to be `wvp * vec4(aPosition, 1)`